notebooks/official/ml_metadata/vertex-pipelines-ml-metadata.ipynb

```diff
@@ -8,7 +8,7 @@
       },
```

Comment:
<p>use notebook template</p><p> </p><p>combine all installs into single cell</p><p> </p><p>Add -q option to pip3</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/826/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>